src/_locales/ru/messages.json

```diff
@@ -60,76 +60,76 @@
-    "html_options_other_enable_sync_warning": { "message": "Включение этого будет перезаписывать настройки во всех других браузерах, зарегистрированных в том же профиле Chrome" },
-    "html_options_suspend_on_low_memory": { "message": "Мгновенно приостанавливается при нехватке системной памяти" },
-    "html_options_suspend_on_low_memory_tooltip_line1": {"message": "Этот параметр будет отменять автоматическую отмену функциональных возможностей браузера Chrome." },
-    "html_options_suspend_on_low_memory_tooltip_line2": {"message": "Если включено, это заставит вкладку приостанавливаться, а не отбрасываться Chrome." },
-    "html_options_suspend_on_low_memory_tooltip_line3": {"message": "Как правило, Chrome будет только отбрасывать вкладки, когда системная память становится очень низкой." },
-    "html_options_suspend_on_low_memory_tooltip_line4": {"message": "Обратите внимание: это может привести к приостановке вкладок раньше, чем у вас есть"},
-    "html_options_suspend_on_low_memory_tooltip_line5": {"message": "указано в параметрах автоматической приостановки табуляции." },
-    "html_options_suspend_on_low_memory_tooltip_line6": {"message": "Для получения дополнительной информации об отбрасывании вкладок Chrome см.:"},
+    "html_options_other_enable_context_menu_tooltip_line2": { "message": "Это даст Вам доступ к дополнительным действиям с вкладками, которые" },
+    "html_options_other_enable_context_menu_tooltip_line3": { "message": "недоступны в другом месте расширения." },
+    "html_options_other_enable_sync": { "message": "Синхронизировать настройки с Вашим профилем Chrome" },
+    "html_options_other_enable_sync_tooltip_line1a": { "message": "Настройки расширения будут синхронизированы с Вашим профилем между" },
+    "html_options_other_enable_sync_tooltip_line1b": { "message": "всеми браузерами Chrome." },
+    "html_options_other_enable_sync_tooltip_line2": { "message": "Также синхронизирует настройки в режиме инкогнито." },
+    "html_options_other_enable_sync_warning": { "message": "Включение этой опции приведет к перезаписыванию настроек расширения для Вашего профиля между всеми браузерами Chrome" },
+    "html_options_suspend_on_low_memory": { "message": "Приостанавливать вкладки при нехватке системной памяти" },
+    "html_options_suspend_on_low_memory_tooltip_line1": {"message": "Этот параметр заменяет встроенный в Chrome механизм выгрузки вкладок." },
+    "html_options_suspend_on_low_memory_tooltip_line2": {"message": "Если включено, вкладка будет принудительно приостанавливаться, вместо использования механизма выгрузки Chrome." },
+    "html_options_suspend_on_low_memory_tooltip_line3": {"message": "Как правило, Chrome выгружает вкладки, только когда системной памяти становится очень мало." },
+    "html_options_suspend_on_low_memory_tooltip_line4": {"message": "Обратите внимание: это может привести к приостановке вкладок раньше, чем выбрано"},
```

Comment:
"чем указано" may be better.

Reply:
+1 here

Reply:
"указано" can mean not only the choice of the user, but also something permanent.
"выбрано" makes it clear that this is a variable parameter.
if you still insist, I will change.

Reply:
Well @MonsterNya also right here coz when user goes to the settings he chooses the time for  "Automatically suspend tabs after" with dropdown menu, not writes it down by himself. And I guess verb specify could also be used here 🤷‍♂️

Reply:
Of course "указать" is correct. it is a matter of choice between "Вы" and "вы". just word "выбор" used most often in programs. 
![image](https://user-images.githubusercontent.com/23438485/52421311-dda5b980-2b04-11e9-8f84-fe5775e2b97a.png)

Reply:
@deanoemcke, which variant in English is more accurate:

> This may cause tabs to suspend earlier than the time you choosed

or

> This may cause tabs to suspend earlier than the time you specified